### PR TITLE
C367997 Verify that "Actions" menu is hidden on the "Logs" tab-- Loca…

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C367997](https://foliotest.testrail.io/index.php?/cases/view/367997)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5716/allure)

image

### CODE REVIEW

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.bulkEditCsvView.gui,
+        Permissions.bulkEditCsvEdit.gui,
+        Permissions.bulkEditLogsView.gui,
+        Permissions.uiUserEdit.gui,
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:19]: does the TC ask for this permission?

Comment (KarapetyanSofia) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:19]: No, but without it, doesnt possible!

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    });
+
+    after('Delete test data', () => {
+      Users.deleteViaApi(user.userId);
+      FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
+    });
+
+    it(
+      'C367997 Verify that "Actions" menu is hidden on the "Logs" tab-- Local approach (firebird) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.firebird] },
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:38]: Update with string

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:13]: wrong directory

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    it(
+      'C367997 Verify that "Actions" menu is hidden on the "Logs" tab-- Local approach (firebird) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+      () => {
+        BulkEditSearchPane.verifySpecificTabHighlighted('Identifier');
+        BulkEditSearchPane.verifyPanesBeforeImport();
+
+        BulkEditSearchPane.verifyRecordIdentifierItems();
+        BulkEditSearchPane.verifyBulkEditPaneItems();
+        BulkEditSearchPane.actionsIsAbsent();
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:45]: Please, ask QA fix TC if
"Actions" menu is enabled at the right upper corner
is wrong assertion

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
+    });
+
+    it(
+      'C367997 Verify that "Actions" menu is hidden on the "Logs" tab-- Local approach (firebird) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+      () => {
+        BulkEditSearchPane.verifySpecificTabHighlighted('Identifier');
+        BulkEditSearchPane.verifyPanesBeforeImport();
+
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js:42]: Please, use empty row to divide TC steps, not randomly

**Pull request merged**